05/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./reset.css">
+    <link rel="stylesheet" media="(max-width:800px)" href="./mobile.css">
+    <link rel="stylesheet" media="(min-width:801px" href="./desktop.css">
+</head>
+<body>
+
+    <header class="short__container">
```

Comment:
Osobiście zrobiłbym 1 klasę `.container`, która zawierałaby wspólne cechy dla wszystkich 3 `__container`.
Dodatkowo jeśli masz element czyli `short__container` (zgodnie z BEM) to trzeba gdzieś wyżej w strukturze osadzić `short`.

PS. Zazwyczaj `.container` to element, który ma określoną maksymalną szerokość i jest wyśrodkowany (u ciebie to `*__row`